sunpy/time/timerange.py

```diff
@@ -26,6 +26,10 @@ class TimeRange:
        Regardless of how a `sunpy.time.TimeRange` is constructed it will always
        provide a positive time range where the start time is before the end time.
 
+       `~sunpy.time.TimeRange.__contains__` has been implented which means you can
```

Comment:
```suggestion
       `~sunpy.time.TimeRange.__contains__` has been implemented which means you can
```

Reply:
Damn it.

Reply:
Fixed. This is what I get for writing english.